app/controllers/application_controller.rb

```diff
@@ -8,11 +8,8 @@ class ApplicationController < ActionController::Base
   before_action :set_sentry_context
 
   def after_sign_in_path_for(resource)
-    if resource.instance_of?(Agent)
-      authenticated_agent_root_path
-    elsif resource.instance_of?(User)
-      stored_location_for(resource) || users_rdvs_path
-    end
+    home_page_when_logged = resource.is_a?(Agent) ? authenticated_agent_root_path : users_rdvs_path
+    stored_location_for(resource) || home_page_when_logged
```

Comment:
Au passage, la fonctionnalité de redirect [décrite ici](https://github.com/heartcombo/devise/wiki/How-To:-Redirect-back-to-current-page-after-sign-in,-sign-out,-sign-up,-update#storelocation-to-the-rescue) n'était pas activée pour les agents, donc je l'ai activée.